Sources/App/CRP/SlackCommand+CRP.swift

```diff
@@ -39,36 +39,21 @@ extension SlackCommand {
                     branch: branch
                 )
 
-                let jiraVersionName = repoMapping.crp.jiraVersionName(release)
-
                 return try github.changelog(for: release, on: container)
                     .catchError(.capture())
-                    .map { ChangelogSection.makeSections(from: $0, for: release) }
-                    .flatMap { (changelogSections: [ChangelogSection]) -> Future<(JiraService.CreatedIssue, [ChangelogSection])> in
-                        // Create CRP Issue
-                        let crpIssue = JiraService.makeCRPIssue(
-                            jiraBaseURL: jira.baseURL,
-                            crpConfig: repoMapping.crp,
+                    .flatMap { (commitMessages: [String]) -> Future<(JiraService.CreatedIssue, JiraService.FixedVersionReport)> in
+                        try jira.executeCRPTicketProcess(
+                            commitMessages: commitMessages,
                             release: release,
-                            changelog: jira.document(from: changelogSections)
+                            repoMapping: repoMapping,
+                            crpProjectID: .init(id: "13402"), // CRP Board
```

Comment:
Would be good to have this as a constant and avoid the comment.

Reply:
Done in 680358a13e1a55e58423b46ab42e27ddb8ac32b6